draft-ietf-oauth-rfc7523bis.xml

```diff
@@ -18,7 +18,7 @@
   <?rfc subcompact="no"?>
 
   <front>
-    <title abbrev="Updates to Audience Values for ASs">Updates to Audience Values for OAuth 2.0 Authorization Servers</title>
+    <title>Updates to OAuth 2.0 JSON Web Token (JWT) Client Authentication and Assertion-Based Authorization Grants</title>
```

Comment:
By accident while looking at some other draft I noticed that the abbrev value does show up in the footer  of the txt version rendered under the other stuff on the main datatracker page https://datatracker.ietf.org/doc/draft-ietf-oauth-rfc7523bis/ so there we should probably consider adding something short and without the word ass.